ZAPD/HighLevel/HLModelIntermediette.cpp

```diff
@@ -310,12 +306,12 @@ string HLModelIntermediette::ToAssimpFile()
 	newScene->mRootNode->mNumChildren += newScene->mNumMeshes;
 	newScene->mRootNode->mChildren = new aiNode*[newScene->mRootNode->mNumChildren];
 
-	for (int i = 0; i < newScene->mNumMeshes; i++)
+	for (int32_t i = 0; i < newScene->mNumMeshes; i++)
```

Comment:
If you are going to change the type of the iterator variables, then you should use `size_t` for them.